src/utils/src/backend-supports/project-supports.ts

```diff
@@ -3,13 +3,24 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-const calculateEndDate = (start: Date, weeks: number) => {
+/**
+ * This function calculates the end date.
+ * @param start the start date
+ * @param weeks number of weeks
+ * @returns the start date after the weeks have passed
+ */
+ const calculateEndDate = (start: Date, weeks: number) => {
   const end = new Date(start);
   end.setDate(start.getDate() + weeks * 7);
   return end;
 };
 
-const projectDurationBuilder = (wps: any) => {
+/**
+ * This symbol function calculates the duration 
+ * @param wps an array of work packages
+ * @returns the duration of the project in weeks
+ */
+const calculateDuration = (wps: any) => {
```

Comment:
You can instead of saying any for the type, set the type to `{ startDate: Date, duration: number }` to describe the minimum type the function works on. A WorkPackage is also of this type since it has a startDate that is a date, and a duration that is a number.

Reply:
Avoiding the any type is best